pkg/controller/storagecluster/reconcile.go

```diff
@@ -461,3 +510,57 @@ func newCephCluster(sc *ocsv1.StorageCluster, cephImage string) *cephv1.CephClus
 
 	return cephCluster
 }
+
+// newStorageClassDeviceSets converts a list of StorageDeviceSets into a list of Rook StorageClassDeviceSets
+func newStorageClassDeviceSets(storageDeviceSets []ocsv1.StorageDeviceSet, topologyMap ocsv1.NodeTopologyMap) []rook.StorageClassDeviceSet {
+	var storageClassDeviceSets []rook.StorageClassDeviceSet
+
+	for _, ds := range storageDeviceSets {
+		topologyKey := ds.TopologyKey
```

Comment:
Does `ds.TopologyKey` exist? TopologyKey of which spec will this pick?

Reply:
It does. Defaults to empty string.